it-and-security/lib/software/mac-mozilla-firefox.yml

```diff
@@ -0,0 +1,2 @@
+url: https://download-installer.cdn.mozilla.net/pub/firefox/releases/132.0.2/mac/en-US/Firefox%20132.0.2.dmg
```

Comment:
D'oh! No DMG support.

Reply:
Use this: https://releases.mozilla.org/pub/firefox/releases/132.0.2/mac/en-US/Firefox%20132.0.2.pkg